src/main/java/org/splitec/service/UvHealthService.java

```diff
@@ -43,13 +47,10 @@ public int findSecureExposureTime(GetIndex.SafeExposureTime exposureTimeResponse
     }
   }
 
-  public int findUserHealthPoints(String username) throws RuntimeException {
-    try {
-      //TODO: Fazer uma busca no banco de dados e recuperar o valor de UVHealthPoints baseado no username
-      return 98;
-    } catch (Exception e) {
-      throw new RuntimeException("User not found");
-    }
+  public int findUserHealthPoints(String username) {
+    CosmosService cosmos = new CosmosService();
+    User user = cosmos.getUser(username);

```

Comment:
@luucasfl Please abstract this to a method

Reply:
Done